CardForm/src/main/res/layout/bt_card_form_fields.xml

```diff
@@ -170,4 +170,20 @@
         android:paddingStart="@dimen/bt_mobile_number_explanation_padding"
         android:paddingEnd="0dp"/>
 
+    <LinearLayout
+        style="@style/bt_field_container"
+        android:paddingRight="0dp"
+        android:paddingLeft="@dimen/bt_mobile_number_explanation_padding"
+        android:paddingStart="@dimen/bt_mobile_number_explanation_padding"
+        android:paddingEnd="0dp">
+
+        <com.braintreepayments.cardform.view.InitialValueCheckBox
+            android:id="@+id/bt_card_form_save_card_checkbox"
+            android:layout_marginLeft="-7dp"
```

Comment:
This should be moved to a dimen value: `@dimen/bt_save_card_checkbox_left_margin`